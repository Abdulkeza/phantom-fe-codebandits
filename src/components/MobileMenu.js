import React from 'react';
import Button from './Button.js';
import LanguageButton from './LanguageButton.js';

const MobileMenu = ({ styles }) => {
  return (
    <div
      className={`${styles} flex flex-col font-raleway md:hidden lg:hidden xl:hidden border px-4 py-2 mr-6`}
    >
      <Button
        name="For operators"
        styles="bg-primary hover:bg-hover text-white rounded-xl mt-2 px-3 py-2"
      />
      <Button
        name="Track Bus"
        styles="border border-primary bg-background hover:bg-hover2 text-primary rounded-xl mt-2 px-6 py-2"
      />
      <LanguageButton className="mt-2" />
    </div>
  );
};

export default MobileMenu;
