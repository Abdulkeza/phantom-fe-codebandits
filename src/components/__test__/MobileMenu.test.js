import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import MobileMenu from '../MobileMenu.js';

describe('MobileMenu', () => {
  it('should render Mobile menu', () => {
    const mobileMenu = renderer
      .create(
        <BrowserRouter>
          <MobileMenu />
        </BrowserRouter>
      )
      .toJSON();
    expect(mobileMenu).toMatchSnapshot();
  });
});
