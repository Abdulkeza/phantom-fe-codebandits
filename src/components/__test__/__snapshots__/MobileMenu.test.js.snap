// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileMenu should render Mobile menu 1`] = `
<div
  className="undefined flex flex-col font-raleway md:hidden lg:hidden xl:hidden border px-4 py-2 mr-6 z-50"
>
  <a
    href="/login"
    onClick={[Function]}
  >
    <div>
      <button
        className="bg-primary hover:bg-hover text-white rounded-xl mt-2 px-3 py-2 rounded-xl py-3 px-5"
        type="button"
      >
        For operators
      </button>
    </div>
  </a>
  <div>
    <button
      className="border border-primary bg-background hover:bg-hover2 text-primary rounded-xl mt-2 px-6 py-2 rounded-xl py-3 px-5"
      type="button"
    >
      Track Bus
    </button>
  </div>
  <div>
    <button
      className="undefined md:mt-8 lg:mt-8 xl:mt-8 flex py-2 px-4 text-lg hover:font-bold"
      onClick={[Function]}
      type="button"
    >
      <h1>
        English
      </h1>
      <img
        alt="dropdown"
        className="w-3 mt-3 ml-2 h-2"
        src="dropdown.png"
      />
    </button>
  </div>
</div>
`;
